pkg/networkservice/chains/nsmgr/heal_test.go

```diff
@@ -538,20 +538,20 @@ func testNSMGRCloseHeal(t *testing.T, withNSEExpiration bool) {
 		SetNSMgrProxySupplier(nil).
 		SetRegistryProxySupplier(nil)
 
-	if withNSEExpiration {
-		builder = builder.SetRegistryExpiryDuration(time.Second / 2)
-	}
-
```

Comment:
Why do we change this?

Reply:
We now rely mostly on the token rather than registryExpirationDuration. Therefore, below we create a token with the expiration we need.

Reply:
Got it